Sources/Extensions/Foundation/DataExtensions.swift

```diff
@@ -16,7 +16,28 @@ public extension Data {
 		// http://stackoverflow.com/questions/38097710/swift-3-changes-for-getbytes-method
 		return [UInt8](self)
 	}
-	
+    
+    /// SwifterSwift: Return data.bytes as a hex string.
+    ///
+    ///     let data = Data(bytes: [0x30, 0x31, 0x32, 0x33])
+    ///     let hexString = data.hexString -> "0123"
+    public var hexString: String {
+        return bytes.reduce("", {
```

Comment:
We probably want to make these methods and not properties because they require an iteration through a bytes array.